TarkovMapOverlay/MainWindow.xaml.cs

```diff
@@ -22,23 +24,119 @@ public partial class MainWindow : Window
         private double opacity;
         private bool transparentBackground = false;
         private Keys minimizeKey = Keys.M;
+        private MouseButtons minimizeButton = MouseButtons.Right;
         private bool toggleMinimizeKeybind = false;
+        private bool toggleMinimizeMousebutton = false;
+        private string currentOpenImagePath;
+        private List<string> _savedMaps = new List<string>();
+
 
         public MainWindow()
         {
             InitializeComponent();
+
             // This ensures that the window is always on top, doesn't always work but should be good enough
             this.Topmost = true;
-
-            // Initial opacity
-            sliderMenu.Value = 100;
+            
+            //load SettingsFile if exists
+            SavedSettings settings = LoadSettings();
+            //load keybinds
+            minimizeKey = settings.minimizeKey;
+            minimizeButton = settings.minimizeMousebutton;
+            minimizeKeybindItem.Header = "_Change " + minimizeKey.ToString() + " Keybind for minimizing";
+            minimizeMouseButtonItem.Header = "_Change " + minimizeButton.ToString() + " Mousebutton for minimizing";
+            //load saved opacity
+            sliderMenu.Value = settings.visual_opacity * 100;
+            //load saved windowState
+            this.Top = settings.windowTop;
+            this.Left = settings.windowLeft;
+            this.Height = settings.windowHeight;
+            this.Width = settings.windowWidth;
+            MoveIntoView();
+            //load last opened Map if file exists
+            if (settings.currentMapPath != null)
+            {
+                try
+                {
+                    BitmapImage bitmap = new BitmapImage(new Uri(settings.currentMapPath, UriKind.RelativeOrAbsolute));
```

Comment:
This line is giving me an error.

```
Argument 2: cannot convert from 'System.UriKind' to 'bool'
```

Reply:
```suggestion
                    BitmapImage bitmap = new BitmapImage(new Uri(settings.currentMapPath, uriKind: UriKind.RelativeOrAbsolute));
```
I think adding `uriKind: ` as an argument fixes this issue.